README.md

```diff
@@ -37,6 +37,30 @@ See [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_gui
 
 See the [changelog](https://github.com/ansible-collections/community.hashi_vault/tree/main/CHANGELOG.rst).
 
+## FAQ
+
+### **Q:** Why not have a single collection of Hashicorp products instead of one just for Vault?
```

Comment:
```suggestion
### **Q:** Why not have a single collection of HashiCorp products instead of one just for Vault?
```
From their [Wikipedia entry](https://en.wikipedia.org/wiki/HashiCorp), that's their correct spelling.